src/components/card/persona-card.ts

```diff
@@ -55,7 +55,7 @@ export class RadixPersonaCard extends LitElement {
         <div class="content">
           <span class="persona">${this.persona}</span>
           <ul>
-            ${this.personaData.map((item) => html`<li>${item}</li>`)}
+            ${(this.personaData || []).map((item) => html`<li>${item}</li>`)}
```

Comment:
`personaData: string[] = []`

PersonaData default value is an empty array. I don't think `this.personaData || []` is needed here

Reply:
I've seen errors in console because of personaData being `null` for some reason. Do we want to preventsuch situations or let it be?

Reply:
I see. Lets keep it then